pkg/virt-launcher/virtwrap/nichotplug_test.go

```diff
@@ -116,6 +120,25 @@ var _ = Describe("nic hotplug on virt-launcher", func() {
 			map[string]api.Interface{},
 			[]v1.Network{generateNetwork(networkName, nadName)},
 		),
+		Entry("given vmi with SR-IOV interfaces, should return no network",
```

Comment:
> given vmi with SR-IOV interfaces, should return no network

Please keep the same format as the other test descriptions.
E.g.:
> vmi with 1 SR-IOV network (when the pod interface *is* ready) and no interfaces in the domain

Reply:
The current format is odd because it doesn't express the expected result: 
`<fn name> given ...`

I will change it to what you suggest.